.gitignore

```diff
@@ -172,3 +172,6 @@ cython_debug/
 
 # PyPI configuration file
 .pypirc
+
+# Database
+bot_user_data.json
```

Comment:
As the path to the database would be customizable, this entry in the `.gitignore` file no longer makes sense. Please go ahead and remove it.